foodstuffs.sk

```diff
@@ -33,11 +33,11 @@ unchanged foods:
 # Before the flattening in 1.13, some foods used data values.
 foods before flattening:
     minecraft version = 1.12.2 or older
-    [raw] (fish¦es|(cod|fish) [fillet]¦s) = minecraft:fish
+    [raw] (fish¦es|cod [fillet]¦s|fish fillet¦s) = minecraft:fish
     [raw] salmon [fillet]¦s = minecraft:fish {Damage:1}
     clownfish¦es = minecraft:fish {Damage:2}
     pufferfish¦es = minecraft:fish {Damage:3}
-    cooked (fish¦es|(cod|fish) [fillet]¦s) = minecraft:cooked_fish
+    cooked (fish¦es|cod [fillet]¦s|fish fillet¦s) = minecraft:cooked_fish

```

Comment:
I made it so you could do just `raw fish` and `cooked fish` here intentionally because that's what the items are called in 1.12 and below; the English display name for cod is simply "fish", which is why fillet was optional (just to let them write it in a more natural way even though the word fillet isn't used in the names of any fish items in any version).

Reply:
I believe you still can since the first part is fish|es

I think every option would be:

cooked fish
cooked cod
cooked cod fillet
cooked fish fillet

Reply:
Ah yeah, I zeroed in on the change and forgot about the first part. Makes sense to me then; I see now that the old syntax had fish twice due to the optional `[fillet]`, which allowed both "fishes" and "fishs" as plural forms. This makes more sense.